dev/.env.docker-compose

```diff
@@ -0,0 +1,13 @@
+DJANGO_CONFIGURATION="DevelopmentConfiguration"
+DJANGO_SETTINGS_MODULE="rdwatch.server.settings"
+RDWATCH_SECRET_KEY="secretkey"
+RDWATCH_POSTGRESQL_URI="postgresql://rdwatch:secretkey@postgresql:5432/rdwatch"
+RDWATCH_REDIS_URI="redis://:secretkey@redis-master:6379"
+RDWATCH_CELERY_BROKER_URL="amqp://rdwatch:secretkey@rabbitmq:5672/"
+RDWATCH_DJANGO_DEBUG=1
+RDWATCH_ACCENTURE_VERSION="2"
+RDWATCH_MINIO_STORAGE_ENDPOINT=minio:9000
+RDWATCH_MINIO_STORAGE_ACCESS_KEY=rdwatch
+RDWATCH_MINIO_STORAGE_SECRET_KEY=secretkey
+RDWATCH_STORAGE_BUCKET_NAME=rdwatch
+AWS_REQUEST_PAYER=requester
```

Comment:
Additionally for local testing probably some example of the environment variables:
`      RDWATCH_SMART_STAC_URL: ${RDWATCH_SMART_STAC_URL:?}
      RDWATCH_SMART_STAC_KEY: ${RDWATCH_SMART_STAC_KEY:?}
`

Reply:
Added .env.private support https://github.com/ResonantGeoData/RD-WATCH/pull/192/commits/cc31451b38dc0cfaab362e35181ba8de17b68419